src/Agent/NewRelic/Agent/Core/Utilization/AzureFunctionAppVendorModel.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using Newtonsoft.Json;
+
+namespace NewRelic.Agent.Core.Utilization
+{
+    public class AzureFunctionVendorModel : IVendorModel
+    {
+        public string VendorName => "azurefunction";
+
+        [JsonProperty("faas.app_name", NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
For vendor data, we normally do not have optional fields.  if some bit of data is not available, we don't send that vendor up.  Per the agent spec:

> If no vendor information is found, the entire vendors hash MUST be excluded. A vendor key MUST be included if all its information was successfully gathered. If a vendor was detected, but the agent could not gather all the expected fields, the key MUST be excluded.